spec/mailers/claim_mailer_spec.rb

```diff
@@ -107,7 +107,15 @@
       let(:mail) { ClaimMailer.rejected(claim) }
 
       it "mentions “the eligible-school during the financial year” reason" do
-        expect(mail.body.encoded).to include("you did not teach at an eligible school between 6 April 2018 and 5 April 2019")
+        expect(policy_configurations(:student_loans).current_academic_year).to eq AcademicYear.new("2025/2026")
```

Comment:
Again, I'm not sure why the academic year is set to be this year. Can we have something that specifically sets it first?